new/PRS13232EhetaMar.xml

```diff
@@ -0,0 +1,61 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13232EhetaMar" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ʾƎḫǝta Māryām</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="SH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="SH" when="2020-11-13">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">እኅተ፡ ማርያም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎḫǝta
+                        Māryām</persName>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13232EhetaMar" passive="MNC010">
+                        <desc>The name of ʾƎḫǝta Māryām is mentioned occasionally in the
+                            supplication formulas throughout the manuscript.</desc>
```

Comment:
Is she mentioned in the main hand? If so, I would add this in `<desc/>`. This also allows to give us a possible terminus for her lifetime: e.g. if the manuscript is dated to the 18th-19th cent. and ʾƎḫǝta Māryām is written in the main hand, I would add `<floruit notBefore="1700" notAfter="1900"/>` within `<person/>`. Also, if she is mentioned in the main hand, I would say something like "She might have been the owner or the donor of the manuscript" within `<desc/>`; if she is mentioned in a secondary hand, I would say then "She might have been a secondary owner or donor of the manuscript".

Reply:
Thank you! I updated the entry.